### ISSUE TITLE
Select buttons aria-expanded attribute always remaining extant, both before and after the button is clicked

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#2985]
### Descriptive summary

From the Accessibility360 audit report:
> The aria-expanded attribute is not assigned until after the button is clicked.
>
> Fix: Mark this button up with the aria-expanded attribute always remaining extant, both before and after the button is clicked.

### Rationale

Screen reader readability.  Improved accessibility.

### Expected behavior
Mark this button up with the `aria-expanded` attribute always remaining extant, both before and after the button is clicked.

### Actual behavior
`aria-expanded` attribute is only applied after button is clicked.

### Occurrences To Fix
- [x] Homepage: "All" select dropdown.  (https://nurax.curationexperts.com/)

![homepage-all-button](https://user-images.githubusercontent.com/3020266/39705257-79d97582-51d3-11e8-8b9a-4d5e1c52ede6.png)

- [x] "Select an action" select dropdown 

![work-item-page](https://user-images.githubusercontent.com/3020266/39704762-00d8a064-51d2-11e8-910d-59cbafd615cc.png)

- [ ] "Limit your search" menu items (Blacklight)

![facets](https://user-images.githubusercontent.com/3020266/39704905-77abcc66-51d2-11e8-863b-8c5f11703fc3.png)

- [ ] "Sort by" and "Per page" select dropdown

![sort-by-relevance-drop-down](https://user-images.githubusercontent.com/3020266/39705141-22b96654-51d3-11e8-81bc-1c73dad0d3ae.png)

**Comment by kdid** [comment]
Reopening to track the Blacklight issues. 

PR for sort and per page widgets: https://github.com/projectblacklight/blacklight/pull/1969

**Comment by kdid** [comment]
PR opened for backport of the two Blacklight issues to 6.x here:

https://github.com/projectblacklight/blacklight/pull/1973

**Comment by rjkati** [comment]
`aria-expanded="false"` attribute is present on all of the buttons listed above when the button is not clicked. When the button is clicked, `aria-expanded="true"`

**Comment by jlhardes** [comment]
@rjkati does your review mean this issue is fixed?

**Comment by rjkati** [comment]
@jlhardes Yes, I think this issue is fixed.

**Comment by jlhardes** [comment]
Thanks that is what I thought but wanted to confirm. This issue is closed.
